src/components/molecules/definition/definition.jsx

```diff
@@ -38,15 +38,11 @@ const Definition = ({ results }) => {
 
```

Comment:
no need for this to be in a render function anymore

Reply:
True! Will update.

Reply:
@arcepaul Done!